test/unit/developer_portal/base_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module DeveloperPortal
+  class BaseControllerTest < ActiveSupport::TestCase
+
+    class TestController < DeveloperPortal::BaseController; end
+
+    class FilterReadOnlyParams < BaseControllerTest
```

Comment:
Another nitpick, perhaps "Test" should be at the end for consistency
```suggestion
    class FilterReadOnlyParamsTest < BaseControllerTest
```

Reply:
hm, what's the purpose of the nested `FilterReadOnlyParams` test?

Reply:
The purpose is to somehow describe that the tests inside that class are for the `filter_read_only_params` method